<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="sv-se">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">a5ac3f823ffdd933e903fb4e719690d783f9c5fe</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.SystemException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">57f5702f54620ed91a29a69805aa82725c892f19</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">70a59b029deacaf8b8b4e8164978853c52628f64</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>Serves as the base class for system exceptions namespace.</source>
          <target state="translated">Fungerar som basklass för undantag-namnområde.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>This class is provided as a means to differentiate between system exceptions and application exceptions.</source>
          <target state="translated">Den här klassen har angetts som ett sätt att skilja mellan system undantag och undantag för programmet.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>It is the base class of such exceptions as &lt;xref:System.ArgumentException&gt;, &lt;xref:System.FormatException&gt;, and &lt;xref:System.InvalidOperationException&gt;.</source>
          <target state="translated">Det är basklassen för sådana undantag som &lt;xref:System.ArgumentException&gt;, &lt;xref:System.FormatException&gt;, och &lt;xref:System.InvalidOperationException&gt;.&lt;/xref:System.InvalidOperationException&gt; &lt;/xref:System.FormatException&gt; &lt;/xref:System.ArgumentException&gt;</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>&gt; <ph id="ph1">[!IMPORTANT]</ph> &gt;  Because SystemException serves as the base class of a variety of exception types, your code should not throw a SystemException exception, nor should it attempt to handle a SystemException exception unless you intend to re-throw the original exception.</source>
          <target state="translated">&gt; <ph id="ph1">[!IMPORTANT]</ph> &gt; Eftersom SystemException fungerar som basklass för olika typer av undantag, din kod inte utlösa ett undantag för SystemException eller bör det försöker hantera ett SystemException undantag om du inte tänker nytt utlösa det ursprungliga undantaget.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>SystemException uses the HRESULT COR_E_SYSTEM, that has the value 0x80131501.</source>
          <target state="translated">SystemException använder HRESULT COR_E_SYSTEM, som innehåller värdet som 0x80131501.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">Initierar en ny instans av den <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klass.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error, such as "A system error has occurred."</source>
          <target state="translated">Standardkonstruktören &lt;xref:System.Exception.Message%2A&gt;-egenskapen för den nya instansen till en tillhandahålls av systemet meddelande som anger vilket fel som "ett systemfel har uppstått."&lt;/xref:System.Exception.Message%2A&gt;</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">Det här meddelandet tar hänsyn till den aktuella systemkulturen.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.SystemException&gt;.</source>
          <target state="translated">I följande tabell visas de inledande egenskapsvärdena för en instans av &lt;xref:System.SystemException&gt;.&lt;/xref:System.SystemException&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|</source>
          <target state="translated">| Egenskapen | Värdet |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| En null-referens (<ph id="ph1">`Nothing`</ph> i Visual Basic). |   | &lt;xref:System.Exception.Message%2A&gt;| Den lokaliserade felsträngen meddelande. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message.</source>
          <target state="translated">Initierar en ny instans av den <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klass med ett angivet meddelande.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>The content of the <ph id="ph1">`message`</ph> parameter should be understandable to the user.</source>
          <target state="translated">Innehållet i den <ph id="ph1">`message`</ph> parameter ska vara förstå för användaren.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Anroparen av den här konstruktorn är behövs för att säkerställa att den här strängen lokaliserade för den aktuella systemkulturen.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.SystemException&gt;.</source>
          <target state="translated">I följande tabell visas de inledande egenskapsvärdena för en instans av &lt;xref:System.SystemException&gt;.&lt;/xref:System.SystemException&gt;</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Egenskapen | Värdet |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| En null-referens (<ph id="ph1">`Nothing`</ph> i Visual Basic). |   | &lt;xref:System.Exception.Message%2A&gt;| Meddelandet felsträng. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>The message that describes the error.</source>
          <target state="translated">Det meddelande som beskriver felet.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">Initierar en ny instans av den <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klass med serialiserade data.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Den här konstruktorn anropas under deserialisering till att fylla Undantagsobjektet som överförs via en dataström.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.</source>
          <target state="translated">Mer information finns i <bpt id="p1">[</bpt>XML och SOAP-serialiseringen<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">Det objekt som innehåller objektdata som serialiserade.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">Detaljerad information om käll- och målservrar.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initierar en ny instans av den <bpt id="p1">&lt;xref href="System.SystemException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klass med ett angivet meddelande och en referens till det interna undantaget som är orsaken till det här undantaget.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Ett undantag som utlöses när ett direkt resultat av ett tidigare undantag kan innehålla en referens till det tidigare undantaget i den &lt;xref:System.Exception.InnerException%2A&gt;egenskapen.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or a null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic) if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Den &lt;xref:System.Exception.InnerException%2A&gt;-egenskap returnerar samma värde som överförs till konstruktorn eller en null-referens (<ph id="ph1">`Nothing`</ph> i Visual Basic) om den &lt;xref:System.Exception.InnerException%2A&gt;egenskapen tillhandahåller inte värdet i det interna undantaget för konstruktorn.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.SystemException&gt;.</source>
          <target state="translated">I följande tabell visas de inledande egenskapsvärdena för en instans av &lt;xref:System.SystemException&gt;.&lt;/xref:System.SystemException&gt;</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Egenskapen | Värdet |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| Referens för inre undantag. |   | &lt;xref:System.Exception.Message%2A&gt;| Meddelandet felsträng. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Felmeddelandet som förklarar orsaken till undantaget.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Undantag som är orsaken till det aktuella undantaget.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not a null reference (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> in Visual Basic), the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Om den <ph id="ph1">`innerException`</ph> parametern är inte en null-referens (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> i Visual Basic), det aktuella undantaget utlöses i en <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> block som hanterar det interna undantaget.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>